internal/client/conditionorc.go

```diff
@@ -0,0 +1,33 @@
+package client
+
+import (
+	"encoding/json"
+
+	"github.com/google/uuid"
+	coapi "github.com/metal-toolbox/conditionorc/pkg/api/v1/types"
+	rivetsCondition "github.com/metal-toolbox/rivets/condition"
+)
+
+func (c *Client) CreateCondition(serverUUID uuid.UUID) error {
```

Comment:
Suggest breaking this into separate methods for the different Conditions, so to begin with have a `CreateConditionInventory()`